client/packages/common/src/ui/components/inputs/DatePickers/BaseDatePickerInput/BaseDatePickerInput.tsx

```diff
@@ -1,13 +1,36 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { DatePicker, DatePickerProps } from '@mui/x-date-pickers';
 import { BasicTextInput } from '../../TextInput/BasicTextInput';
 import { StandardTextFieldProps, TextFieldProps } from '@mui/material';
 import { useAppTheme } from '@common/styles';
+import { DateUtils } from '@common/intl';
+import { useDebounceCallback } from '@common/hooks';
 
 export const BaseDatePickerInput: FC<
   Omit<DatePickerProps<Date>, 'renderInput'>
 > = props => {
   const theme = useAppTheme();
+  const [internalValue, setInternalValue] = useState<
+    Date | string | number | null
+  >();
+
+  useEffect(() => {
+    if (props.value) setInternalValue(props.value);
+  }, [props.value]);
```

Comment:
Think you have to remember the time part from the props and later merge them back together...